js/lib/danoni_constants.js

```
@@ -2851,83 +2869,71 @@ const g_shortcutObj = {
-        ShiftLeft_Digit3: { id: `lnkfastSlowR` },
-        ShiftRight_Digit3: { id: `lnkfastSlowR` },
-        ShiftLeft_Digit4: { id: `lnklifeGaugeR` },
-        ShiftRight_Digit4: { id: `lnklifeGaugeR` },
-        ShiftLeft_Digit5: { id: `lnkscoreR` },
+        ShiftLeft_Digit3: { id: `lnklifeGaugeR` },
+        ShiftRight_Digit3: { id: `lnklifeGaugeR` },
+        ShiftLeft_Digit4: { id: `lnkscoreR` },
         ShiftRight_Digit5: { id: `lnkscoreR` },
-        ShiftLeft_Digit6: { id: `lnkmusicInfoR` },
+        ShiftLeft_Digit5: { id: `lnkmusicInfoR` },
         ShiftRight_Digit6: { id: `lnkmusicInfoR` },
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)